src/main/java/com/metamx/common/parsers/JSONPathParser.java

```diff
@@ -225,7 +235,7 @@ private boolean isFlatList(List<Object> list)
 
   /**
    * Specifies a field to be added to the parsed object Map, using JsonPath notation.
-   *
+   * <p/>
```

Comment:
preferences->editor->code style->java->javaDoc->generate `<p/>` on empty lines 

that's the one I know about